sdk/storage/test/shares/share_file_client_test.hpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "share_directory_client_test.hpp"
+#include "shares/shares.hpp"
+#include "test_base.hpp"
+
+namespace Azure { namespace Storage { namespace Test {
+
+  class FileShareFileClientTest : public FileShareDirectoryClientTest {
+  protected:
+    static void SetUpTestSuite();
+    static void TearDownTestSuite();
+
+    static std::shared_ptr<Files::Shares::FileClient> m_fileClient;
+    static std::string m_fileName;
+    static std::vector<uint8_t> m_fileContent;
+
+    static void RandomizeContent();
```

Comment:
how is this different from https://github.com/Azure/azure-sdk-for-cpp/blob/70eeec598496de78ba5f6a5ccd7bdb4b4724fd87/sdk/storage/test/test_base.hpp#L57

Reply:
The memory is reused.

Reply:
then how about https://github.com/Azure/azure-sdk-for-cpp/blob/70eeec598496de78ba5f6a5ccd7bdb4b4724fd87/sdk/storage/test/test_base.hpp#L56?

with just one reinterpret_cast

Reply:
I'm OK with it.